pkg/statsd/statsd_defaults_and_params.go

```diff
@@ -62,6 +62,8 @@ const (
 	DefaultHeartbeatInterval = time.Duration(0)
 	// DefaultReceiveBatchSize is the number of packets to read in each receive batch
 	DefaultReceiveBatchSize = 50
+	// DefaultConnPerReader is the default for whether to create a connection per reader
+	DefaultConnPerReader = false
```

Comment:
note I've left this disabled by default - mainly due to concerns about the portability of socket reuse settings